functions/setupApi.js

```diff
@@ -66,24 +66,25 @@ const setupApi = ({ autopilotapikey, middlewares = [] }) => {
       contact: {
         FirstName: name,
         Email: email,
-        LeadSource: '1-day workshops form',
+        LeadSource: pathname,
         _autopilot_list: 'contactlist_37B9CE06-F48D-4F7B-A119-4725B474EF2C',
         custom,
+        'boolean--Resources-SingedUp': resources,
```

Comment:
@ric9176 should we create a const since we use that string in a few places? 
```
contact: {
    // ...
    [BOOLEAN__RESOURCES_SIGNEUP]: resources 
   // heads-up there is a type in SingeUp but I don't know what's the implication of changing it now for the marketing tools 
}
```

Reply:
:+1: